zulip_bots/zulip_bots/lib.py

```diff
@@ -68,7 +68,28 @@ def show_error_and_exit(self) -> None:
         sys.exit(1)
 
 
-class StateHandler:
+class BotIdentity:
+    def __init__(self, name: str, email: str) -> None:
+        self.name = name
+        self.email = email
+        self.mention = '@**' + name + '**'
+
+class BaseStorage(Protocol):
+    def put(self, key: Text, value: Any) -> None:
+        ...
+
+    def get(self, key: Text) -> Any:
+        ...
+
+    def contains(self, key: Text) -> bool:
+        ...
+
+class BotStorage(BaseStorage):
+    @contextmanager
+    def use(self, keys: List[Text]) -> Iterator[BaseStorage]:
+        ...
+
+class StateHandler(BotStorage):
```

Comment:
I don't think `StateHandler` needs to import `BotStorage`

Reply:
If I don't do that, there will be a mypy error.